### PR TITLE
Add `spoc merge` Command

### PR DESCRIPTION
**Pull request by mhils**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

This PR adds a `spoc merge` command which can be used to merge multiple security profiles via the CLI.

#### Does this PR have test?

Yes.

#### Does this PR introduce a user-facing change?

```release-note
Added a `spoc merge` command to merge multiple security profiles from the command line.
```

### CODE REVIEW

File: internal/pkg/cli/merger/consts.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2023 The Kubernetes Authors.
```
Comment (saschagrunert) [inline internal/pkg/cli/merger/consts.go:2]: nit
```suggestion
Copyright 2024 The Kubernetes Authors.
```

the same applies to other new files.

File: internal/pkg/cli/merger/merger.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	contents := make([]client.Object, len(p.options.inputFiles))
+	for i, filepath := range p.options.inputFiles {
+		log.Printf("Reading file %s", filepath)
+		content, err := p.ReadFile(filepath)
+		if err != nil {
+			return fmt.Errorf("open profile: %w", err)
+		}
+
+		// yaml.Unmarshal happily takes YAML for a SELinux profile and unmarshals
+		// it into SeccompProfile. We need to check the YAML kind!
```
[the remark this answers is not in the document]
Comment (saschagrunert) [reply internal/pkg/cli/merger/merger.go:60]: We only used if for seccomp right now, so yeah. This may be a bad bug.

**Review by saschagrunert — COMMENTED** [review]
Thank you!

**Review by saschagrunert — APPROVED** [review]
Thank you!

**Comment by mhils** [comment]
Thank you for the prompt review! 🍰  Tests are just flaky here, right?

**Comment by saschagrunert** [comment]
Yes, restarted the OLM test.

**Comment by mhils** [comment]
Thanks - looks like we are extra lucky today or we have CI breakage. 🙃

**Comment by mhils** [comment]
Thank you for the super quick review! 🍰 ⚡

**Comment by saschagrunert** [comment]
Thank you for working on this @mhils! :pray:

**Pull request merged**